streams/src/test/java/org/apache/kafka/streams/StreamsBuilderTest.java

```diff
@@ -285,7 +286,28 @@ public void shouldProcessViaThroughTopic() {
         assertEquals(Collections.singletonList(new KeyValueTimestamp<>("A", "aa", 0)), sourceProcessorSupplier.theCapturedProcessor().processed);
         assertEquals(Collections.singletonList(new KeyValueTimestamp<>("A", "aa", 0)), throughProcessorSupplier.theCapturedProcessor().processed);
     }
-    
+
+    @Test
+    public void shouldProcessViaRepartitionTopic() {
```

Comment:
Replicated the test for `through()` for `repartition()`.

Reply:
Thanks!